src/foraging_gui/Foraging.py

```diff
@@ -628,21 +638,16 @@ def _check_drop_frames(self,save_tag=1):
                 elif len(os.listdir(video_folder)) == 0:
                     # no video data saved.
                     self.trigger_length=0
-                    self.WarningLabelCamera.setText('')
-                    self.WarningLabelCamera.setStyleSheet(self.default_warning_color)
                     self.to_check_drop_frames=0
                     return
                 elif ('HighSpeedCamera' in self.SettingsBox) and (self.SettingsBox['HighSpeedCamera'] ==1):
                     self.trigger_length=0
                     logging.error('Saved video data, but no camera trigger file found')
-                    self.WarningLabelCamera.setText('No camera trigger file found!')
-                    self.WarningLabelCamera.setStyleSheet(self.default_warning_color)
```

Comment:
![image](https://github.com/user-attachments/assets/27d2fabe-a592-4d61-bb4b-e019baa2ba67)

Reply:
This photo of WarningLabelCamera should sufficiently depict what WarningLabelCamera looks likes, even though different text is assigned to it.